app/src/main/java/com/sunlightlabs/congress/models/Roll.java

```diff
@@ -89,7 +90,7 @@ public static Roll splitRollId(String roll_id) {
 	
 	// formattedNumber can be anything that ends with a number - the number will be extracted,
 	// the chamber's first letter will be used, and combined into a roll ID
-	public static String normalizeRollId(String chamber, String year, String formattedNumber) {
+	public static String normalizeRollId(String chamber, String formattedNumber, String year) {
```

Comment:
Not sure how this wasn't causing problems before, but this was formatting roll ID's backwards. (Or perhaps I changed it elsewhere and didn't update this part.)